### PR TITLE
feat(nfts): add in asset page init styling

### PR DESCRIPTION
**Pull request by rsteubs-bc**
## Description (optional)
asset page init styling

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/graphql/queries/NftAsset.graphql [hunk 1]
```diff
@@ -1,15 +1,94 @@
...
+      seller_fee_basis_points
+    }
+    contract_address
+    collection_slug
+    owner_address
+    creator_address
+    asset_bundle_slug
+    transfer_fee_payment_token_id
+    listing_date
+    date_ingested
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/graphql/queries/NftAsset.graphql:86]: We don't need all these fields, we should only request what we need

File: packages/blockchain-wallet-v4-frontend/src/data/components/nfts/slice.ts [hunk 2]
```diff
@@ -247,6 +248,22 @@ const nftsSlice = createSlice({
...
+      action: PayloadAction<{
+        address: string
+        token_id: string
+      }>
+    ) => {},
+    fetchOpenseaAssetFailure: (state, action: PayloadAction<NftAsset>) => {
+      state.openSeaStatus = Remote.Failure(action.payload)
+    },
+    fetchOpenseaAssetLoading: (state) => {
+      state.openSeaStatus = Remote.Loading
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/data/components/nfts/slice.ts:262]: state.openseaAsset =

File: packages/blockchain-wallet-v4-frontend/src/data/components/nfts/slice.ts [hunk 3]
```diff
@@ -247,6 +248,22 @@ const nftsSlice = createSlice({
...
     },
+    fetchOpenseaAsset: (
+      state,
+      action: PayloadAction<{
+        address: string
+        token_id: string
+      }>
+    ) => {},
+    fetchOpenseaAssetFailure: (state, action: PayloadAction<NftAsset>) => {
+      state.openSeaStatus = Remote.Failure(action.payload)
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/data/components/nfts/slice.ts:259]: state.openseaAsset =

File: packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Asset/index.tsx [hunk 4]
```diff
@@ -1,36 +1,715 @@
...
+const CollectionName = styled.div`
+  padding-bottom: 1em;
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 16px;
+  line-height: 150%;
+  display: flex;
+  align-items: left;
+  color: #121d33;
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Asset/index.tsx:89]: import colors from constellation

color: ${colors.grey900}

File: packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Asset/index.tsx [hunk 5]
```diff
@@ -1,36 +1,715 @@
...
+const AssetName = styled.div`
+  height: 20px;
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 40px;
+  line-height: 125%;
+  display: flex;
+  align-items: left;
+  color: #121d33;
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Asset/index.tsx:101]: import colors from constellation

color: ${colors.grey900}

File: packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Asset/index.tsx [hunk 6]
```diff
@@ -1,36 +1,715 @@
...
+const PriceHistoryTitle = styled(AssetName)`
+  font-size: 24px;
+  line-height: 135%;
+  padding: 2em 0em 2em 0em;
+`
+
+const PriceHistory = styled(PriceHistoryTitle)`
+  font-size: 14px;
+  height: 340px;
+  background: #f0f2f7;
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Asset/index.tsx:113]: import colors from constellation

color: ${colors.XYZ}

File: packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Asset/index.tsx [hunk 7]
```diff
@@ -1,36 +1,715 @@
...
+  padding: 2em;
+  border: 1px solid #dfe3eb;
+  box-sizing: border-box;
+  border-radius: 8px;
+`
+
+const CurrentPriceBox = styled.div`
+  border: 1px solid #dfe3eb;
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Asset/index.tsx:121]: import colors from constellation

File: packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Asset/index.tsx [hunk 8]
```diff
@@ -1,36 +1,715 @@
...
+  const [asset] = useAssetQuery({
     variables: { filter: { contract_address: contract, token_id: id } }
   })
-
-  return <div>{JSON.stringify(results)}</div>
+  const [assets] = useAssetsQuery({
+    variables: { filter: { contract_address: contract } }
+  })
+  const [Tab, setTab] = useState('details')
+  const NULL_ADDRESS = '0x0000000000000000000000000000000000000000'
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Asset/index.tsx:293]: NULL_ADDRESS already defined in @core package somewhere. Import from there.

const WETH_ADDRESS = window.coins.WETH.type.erc20Address

File: packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Asset/index.tsx [hunk 9]
```diff
@@ -1,36 +1,715 @@
...
+  const WETH_ADDRESS = '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2'
+  useEffect(() => {
+    nftsActions.fetchOpenseaAsset({
+      address: asset?.data?.asset?.contract_address || '0xbc4ca0eda7647a8ab7c2061c2e118a18a936f13d',
+      token_id: asset.data?.asset?.token_id || '8520'
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Asset/index.tsx:298]: Hmm maybe lets fail and display a 404 type page?

File: packages/blockchain-wallet-v4/src/network/api/nfts/index.ts [hunk 10]
```diff
@@ -146,6 +146,14 @@ export default ({ apiUrl, get, openseaApi, post }) => {
     })
   }
 
+  const getOpenSeaAsset = (collection_id: string, asset_number: string): NftAsset => {
+    return get({
+      endPoint: `asset/${collection_id}/${asset_number}`,
+      ignoreQueryParams: true,
+      url: `https://api.opensea.io/api/v1/`
```
Comment (plondon) [inline packages/blockchain-wallet-v4/src/network/api/nfts/index.ts:153]: If you merge in latest changes you should see `openseaUrl`

**Review by plondon — CHANGES_REQUESTED** [review]
Havent pulled it down too look at it but just went through the code and found some small issues:

1. copy/pasta error in slice
2. import colors from constellation
3. other small fixes

Lmk when I can look again and Ill pull down!

**Review by Marek0491 — APPROVED** [review]
https://github.com/blockchain/blockchain-wallet-v4-frontend/tree/feat/opensea-graphql

File: packages/blockchain-wallet-v4-frontend/src/data/components/nfts/sagas.ts [hunk 11]
```diff
@@ -144,6 +144,19 @@ export default ({ api }: { api: APIType }) => {
     }
   }
 
+  const fetchOpenseaAsset = function* (action: ReturnType<typeof A.fetchOpenseaAsset>) {
+    try {
+      yield put(A.fetchNftOffersMadeLoading())
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/data/components/nfts/sagas.ts:149]: fetchOpenseaAssetLoading()

File: packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Asset/index.tsx [hunk 12]
```diff
@@ -1,36 +1,726 @@
...
-  const [results] = useAssetQuery({
+  // @ts-ignore
+  const [asset] = useAssetQuery({
     variables: { filter: { contract_address: contract, token_id: id } }
   })
+  const [assets] = useAssetsQuery({
+    variables: { filter: { contract_address: contract } }
+  })
+  const [Tab, setTab] = useState('details')
+  const NULL_ADDRESS = '0x0000000000000000000000000000000000000000'
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Asset/index.tsx:295]: import NULL_ADDRESS from @core

Comment (plondon) [reply packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Asset/index.tsx:295]: import { NULL_ADDRESS } from '@core/redux/payment/nfts/utils'

**Review by plondon — APPROVED** [review]
A few more small things and then Ill merge!

**Pull request merged**